code/game/objects/objs.dm

```diff
@@ -220,7 +220,8 @@ GLOBAL_LIST_EMPTY(objects_by_id_tag)
 	SEND_SIGNAL(source, COMSIG_REAGENTS_EXPOSE_OBJ, src, reagents, methods, volume_modifier, show_message)
 	for(var/reagent in reagents)
 		var/datum/reagent/R = reagent
-		. |= R.expose_obj(src, reagents[R])
+		var/reac_volume = reagents[R]
+		. |= R.expose_obj(src, methods, reac_volume, show_message)
```

Comment:
```suggestion
		. |= R.expose_obj(src, methods, reagents[R], show_message)
```

Reply:
I deliberately added that var so that we know that reac_volume is being passed into the arg since it's not apparent. I ran into the problem of getting mixed up with volume_modifier since it's not clear that reagents[R] is used for volume.

Reply:
I also should point out we are renaming an ambiguous one letter var.